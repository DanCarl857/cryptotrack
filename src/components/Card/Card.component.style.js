import { StyleSheet } from 'react-native'
import theme from './../../styles/theme.style'

export default StyleSheet.create({
    containerStyle: {
        backgroundColor: theme.WHITE,
        borderWidth: 1,
        borderColor: '#f7f7f7',
        borderBottomWidth: 1,
        borderRadius: 10,
        marginLeft: 15,
        marginRight: 15,
        marginTop: 15,
        padding: 15,
        alignSelf: 'stretch',
        justifyContent: 'center'
      }
})