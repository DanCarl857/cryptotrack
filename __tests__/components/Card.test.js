import React from 'react'
import { View, Text } from 'react-native'
import { shallow } from 'enzyme'
import Card from './../../src/components/Card/Card.component'

describe('Button', () => {
    describe('Rendering', () => {
        it('should match snapshot', () => {
            const component = shallow(
            <Card>
                <View>
                    <Text>Testing the card</Text>
                </View>
            </Card>)
            expect(component).toMatchSnapshot()
        })
    })
})