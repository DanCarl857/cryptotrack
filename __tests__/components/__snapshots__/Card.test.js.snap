// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button Rendering should match snapshot 1`] = `ShallowWrapper {}`;
